packages/react/src/components/Search/next/Search.js

```diff
@@ -135,6 +135,7 @@ const Search = React.forwardRef(function Search(
         className={clearClasses}
         disabled={disabled}
         onClick={clearInput}
+        title="Clear"
```

Comment:
One idea for this field could be to use `closeButtonLabelText` which is currently being used to label the `<button>`.

```suggestion
        title={closeButtonLabelText}
```

One thing we would want to make sure is that `title` is configurable (for example, someone could specify an alternate title) given that people could use this component in different situations or would like to localize this text 👍

Reply:
Hi @joshblack I have updated the title to title={closeButtonLabelText} and the test cases. Thanks for your recommendation.